embedded-aerospike/src/main/java/com/playtika/test/aerospike/AerospikeTestOperations.java

```diff
@@ -74,4 +77,18 @@ private void timeTravel(DateTime newNow) {
         DateTimeUtils.setCurrentMillisFixed(newNow.getMillis());
         expiredDocumentsCleaner.cleanExpiredDocumentsBefore(newNow.getMillis());
     }
+
+    /**
+     * More at https://www.aerospike.com/docs/guide/scan.html
+     * @return current count of performed scans on aerospike server instance.
+     */
+    @SneakyThrows
+    public int getScansCount() {
+        Container.ExecResult execResult = aerospikeContainer.execInContainer("asinfo", "-v", "'scan-list'");
```

Comment:
scans are related to namespace and set.
Could you please migrate these operations here instead: https://github.com/aerospike-community/spring-data-aerospike/blob/master/src/test/java/org/springframework/data/aerospike/BlockingAerospikeTestOperations.java

Reply:
nice, added a couple of additional methods (only related to scans)